lib/middleman-cdn/commands.rb

```diff
@@ -22,23 +22,17 @@ def self.exit_on_failure?
         true
       end
 
-      desc "cdn:cdn_invalidate", "Invalidate your CloudFlare or CloudFront cache"
       def cdn_invalidate(*args)
         begin
-          if args.first && args.first.respond_to?(:filter)
-            options = args.first
-            files = args.drop(1)
+          options, files = if args.first && args.first.respond_to?(:filter)
+            [args.first, args.drop(1)]
           else
-            files = args
+            [Middleman::CDN::Extension.options, args]
           end
 
           if options.nil?
-            app_instance = ::Middleman::Application.server.inst
-            unless app_instance.respond_to?(:cdn_options)
-              self.class.say_status(nil, ANSI.red{ "Error: You need to activate the cdn extension in config.rb.\n#{example_configuration}" })
-              raise
-            end
-            options = app_instance.cdn_options
+            self.class.say_status(nil, ANSI.red{ "Error: You need to activate the cdn extension in config.rb.\n#{example_configuration}" })
```

Comment:
How does the `options.nil?` determine if there is config in config.rb without the checks being removed above?